src/Http/Controllers/AutoDocController.php

```diff
@@ -18,9 +19,47 @@ public function __construct()
 
     public function documentation()
     {
-        $documentation = $this->service->getDocFileContent();
+        $this->documentation = json_decode(json_encode($this->service->getDocFileContent()), true);
```

Comment:
```suggestion
        $this->documentation = json_decode($this->service->getDocFileContent(), true);
```

Reply:
As we discussed let's update the interface to change the type of getDocFileContent method of drivers to array instead of std